scd30/scd30.c

```diff
@@ -88,8 +88,9 @@ int16_t scd30_read_measurement(float32_t *co2_ppm, float32_t *temperature,
         sensirion_i2c_write_cmd(SCD30_I2C_ADDRESS, SCD30_CMD_READ_MEASUREMENT);
     if (ret != STATUS_OK)
         return ret;
-    ret =
-        sensirion_i2c_read_bytes(SCD30_I2C_ADDRESS, data->bytes, sizeof(data));
+
+    ret = sensirion_i2c_read_words_as_bytes(SCD30_I2C_ADDRESS, data->bytes,
```

Comment:
Is this function part of the public API? If yes we should add the rename to the CHANGELOG and it would be a breaking change.

Reply:
The function could of course be used publicly, but I don't consider it a public interface method since it's only useful to read from our sensors (b/c word-size, CRC) - the interface to the sensor is the driver itself.

Reply:
So then it is not part of the public API of the driver and hence not a breaking change.